sdk/src/vote_program.rs

```diff
@@ -203,24 +269,13 @@ mod tests {
     fn test_vote_serialize() {
         let mut buffer: Vec<u8> = vec![0; get_max_size()];
         let mut vote_state = VoteState::default();
-        vote_state.votes.resize(MAX_VOTE_HISTORY, Vote::default());
+        vote_state
+            .votes
+            .resize(MAX_VOTE_HISTORY, StoredVote::default());
         vote_state.serialize(&mut buffer).unwrap();
         assert_eq!(VoteState::deserialize(&buffer).unwrap(), vote_state);
     }
 
-    #[test]
-    fn test_vote_credits() {
```

Comment:
Rewards program depends on this

Reply:
hehe I kept the test around, its just at the bottom of the file now xD